client/state/ui/selectors.js

```diff
@@ -90,3 +91,39 @@ export function isPreviewShowing( state ) {
 export function getPreviewUrl( state ) {
 	return state.ui.currentPreviewUrl;
 }
+
+/**
+ * Returns the current layout focus area
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}  The current layout focus area
+ */
+export function getCurrentLayoutFocus( state ) {
+	// Temporary while we transition from the Flux store
+	return layoutFocusModule.getCurrent();
```

Comment:
This is where I think it'd be nice if we were to have `layoutFocus` update Redux state so that we can always rely on our state, rather than call on `layoutFocus` from the reducer.

See [`lib/screen-title`](https://github.com/Automattic/wp-calypso/blob/master/client/lib/screen-title/index.js) as an example (looks like it was refactored recently into a middleware in #6669 by @yurynix).